src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -77,6 +72,13 @@ def contents
         )
       end
 
+    private
+
+      def dm_name_for(device)
```

Comment:
Well it is really a copy&paste from Encrypter that was also the only method which make me depend on it so I removed the encrypter part and just obtain the name.

I will add a FIXME to it, and we have been talking already about it.